chat-server/src/routes/conversations/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { Conversation } from "../../services/conversations";
+
+export interface MessageResponse {
+  id: string;
+  role: string;
+  content: string;
+  rating?: boolean;
+  timeCreated: number;
```

Comment:
```suggestion
  createdAt: number;
```

We're using `createdAt` (and `updatedAt` etc) in the spec